guild/_test.py

```diff
@@ -478,6 +474,8 @@ def _check_prompt_blank(lines, indent, name, lineno):
 
 
 class BashDocTestChecker(doctest.OutputChecker):
+    __test__ = False
```

Comment:
These are pytest hints, and they can go away. It's better that pytest just skip this file entirely.